docs/guides/node-operator-manual.md

```diff
@@ -276,3 +326,43 @@ docker run --rm -it \
   --datadir /root/.lighthouse/data \
   --directory /root/validator_keys
 ```
+
+### Checking the keys of all Lido Node Operators
+
+After keys are submitted, it's critical to first wait for the key submission transactions to be mined and after that check all Lido keys for validity.
+
+Never vote for increasing the key limits of Node Operators before verifying new keys are present and valid.
+
+#### Lido CLI
+
+Make sure Python with pip is installed and then run:
+
+```sh
+pip install lido-cli
+lido-cli --rpc http://1.2.3.4:8545 validate_network_keys
+```
+
+This operation checks all Lido keys for validity. This is a CPU-intensive process, for example, a modern desktop with 6 cores, 12 threads and great cooling processes 1k keys in ~1 minute.
+
+You would need an RPC endpoint - a local node / RPC provider (eg Alchemy/Infura).
+
+#### Lido Node Operator Dashboard
+
+You can also check the uploaded keys on [Mainnet Lido Node Operator Dashboard] or [Testnet Lido Node Operator Dashboard].
+
+It will display key summary for each node operator, including checks for number of keys over staking limit and invalid keys if found.
```

Comment:
"The UI show number of submitted, approved and valid keys for each Node Operator, along with all invalid keys in case there are any."